### ISSUE TITLE
Angular: EventEmitter issue with StrictTemplates set to true

### ISSUE THREAD

**Issue by dexster** [ionic-team/stencil-ds-output-targets#155]
Using Angular v12, when setting strictTemplates: true in tsconfig I get this error

`error TS2339: Property 'subscribe' does not exist on type 'EventEmitter<SbgFileUploadChangeEvent>`

The HTML causing the issue looks something like this 
`<my-comp (fileChange)="doSomething()"</my-comp> `

Taking a look at my component during runtime I can see that subscribe is there (see image below)

![Screenshot 2021-05-24 at 08 22 05](https://user-images.githubusercontent.com/6392334/119306112-fbfa3980-bc69-11eb-81ba-3e027dcd470f.png)

Also, If I change the import of EventEmitter in the generated d.ts file for my component from stencil-public-runtime to angular/core this error goes away. Obviously this is not a viable solution though.

The generated proxy is shown below:

```
@ProxyCmp({
    inputs: ['allowedFileTypes', 'disabled', 'fileUploadStatus', 'maxCombinedFileSize', 'maxFileSize', 'multiple', 'required', 'sbgFileUploadStatus'],
    methods: ['reset', 'getSbgFileStatus']
})
@Component({
    selector: 'sbg-file-upload',
    changeDetection: ChangeDetectionStrategy.OnPush,
    template: '<ng-content></ng-content>',
    inputs: ['allowedFileTypes', 'disabled', 'fileUploadStatus', 'maxCombinedFileSize', 'maxFileSize', 'multiple', 'required', 'sbgFileUploadStatus'],
    outputs: ['fileChange']
})
export class SbgFileUploadComponent {
    @Input() disabled: any;

    fileChange!: ISbgFileUpload['fileChange'];
    protected el: HTMLElement;

    constructor(c: ChangeDetectorRef, r: ElementRef, protected z: NgZone) {
        c.detach();
        this.el = r.nativeElement;
        proxyOutputs(this, this.el, ['fileChange']);
    }
}
```

**Comment by javo2223** [comment]
We got the same error when using angular v12?

`Property 'subscribe' does not exist on type 'EventEmitter<any>'.`

Would be very helpful if this bug could be fixed. Our customers can't really use our components in angular v12 applications at the moment.

**Comment by willmartian** [comment]
This issue has been fixed and is available in v0.1.0 of the Angular output target. Thanks for the feedback everyone and thanks to @poulpe38000 for heading the PR contribution.
